boards/aithinker/ai_m62_12f/ai_m62_12f.dts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2024-2025 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <bflb/bl616.dtsi>
+#include "ai_m62_12f-pinctrl.dtsi"
+
+/ {
+	model = "Ai-Thinker M62-12F development board";
+	compatible = "bflb,bl616";
+
+	chosen {
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,itcm = &sram1;
+		zephyr,sram = &sram0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+	};
+};
+
+&cpu0 {
+	clock-frequency = <DT_FREQ_M(320)>;
+};
+
+&flashctrl {
+	flash0: flash@A0000000 {
+		compatible = "soc-nv-flash", "gd,25lq32d";
+		reg = <0xA0000000 (0x400000 - 0x2000)>;
+		write-block-size = <256>;
+		erase-block-size = <DT_SIZE_K(4)>;
+		/* jedec-id = [5e 40 16]; */
```

Comment:
line can go?

Reply:
It's for documentation purposes, soc-nv-flash doesnt take jedec-id, and i dont know of a solution to have a jedec flash to behave the same as soc-nv-flash within zephyr (in this case it's the same thing, but it's a separate die flash in the chip that is a jedec-flash, it can also be external, but in both cases it goes through a controller which settings arent dependent on the jedec flash properties but rather a database of flash linking settings to a flash jedec ID that is provided by the flash tool).

Reply:
Also double checking this is the wrong flash entirely anyway... supposed to be c86016, not sure where i got that gd flash from.

Reply:
I guess in the end you are right, this should be removed because this is one more place to make a mistake, the name is correct but that id is wrong... Will remove it.

Reply:
so, done.